<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>102</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>300</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>文件解密</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2" columnstretch="1,0,0" columnminimumwidth="1,0,0">
   <item row="0" column="0" colspan="3">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>文件：</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="encrypt_file_le"/>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="encrypt_file_btn">
       <property name="text">
        <string>选择文件</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>私钥：</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="prikey_le"/>
     </item>
     <item row="1" column="2">
      <widget class="QPushButton" name="prikey_btn">
       <property name="text">
        <string>选择私钥</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="tip_lb">
     <property name="text">
      <string>提示标签</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QPushButton" name="decrypt_btn">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>解密</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QPushButton" name="path_btn">
     <property name="toolTip">
      <string>打开解密文件目录</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../../mmx_img.qrc">
       <normaloff>:/btn/resource/img/open folder yellow.ico</normaloff>:/btn/resource/img/open folder yellow.ico</iconset>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../mmx_img.qrc"/>
 </resources>
 <connections/>
</ui>
